package.json

```diff
@@ -115,7 +115,7 @@
       "@semantic-release/git",
       "@semantic-release/github"
     ],
-    "branch": "master"
+    "branch": "main"
```

Comment:
at the moment semantic release doesn't support main, you can override their configurations by adding this https://github.com/americanexpress/json-parse-context/pull/2/files

Reply:
Good call! I will update

Reply:
updated (f3ef0e9)